python/.gitignore

```diff
@@ -0,0 +1,18 @@
+.idea/
+*.iml
+venv/
+release.properties
+pom.xml.releaseBackup
```

Comment:
Please remove irrelevant entries from this file

Reply:
Apologies - I popped in my local gitignore that I use as a standard template.  Will tidy up